packages/internal-playwright-helpers/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@inrupt/internal-playwright-helpers",
+  "version": "1.4.2",
+  "description": "This package provides page models for known common elements of the sdk testable UIs",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "build": "rollup -c rollup.config.mjs"
+  },
+  "author": "Inrupt <devtools-team@inrupt.com>",
+  "license": "MIT",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js"
+    }
+  },
+  "dependencies": {
+    "@inrupt/internal-playwright-testids": "^1.4.2",
+    "@inrupt/internal-test-env": "^1.4.2"
```

Comment:
Oh interesting, but is this how it should be done even after these ship? I assumed Lerna was the one taking the wheel here.

Reply:
Off top of head, I'm honestly not sure. I just know elsewhere we're using file: links instead of fixed versions & that the publish resolves those to correct versions